### ISSUE TITLE
bug: Tabs cannot be selected via Enter/Space keyboard navigation

### ISSUE THREAD

**Issue by afc163** [ant-design/ant-design#53691]
重现链接：https://ant.design/components/tabs-cn/
版本：5.24.8
被这里改挂了：https://github.com/react-component/tabs/pull/790
